robosuite/renderers/mjviewer/mjviewer_renderer.py

```diff
@@ -1,15 +1,28 @@
+import cv2
+import numpy as np
 from mujoco import viewer
 
 
 class MjviewerRenderer:
-    def __init__(self, env, camera_id=None, cam_config=None):
+    def __init__(self, env, camera_id=None, cam_config=None, sim=None):
         self.env = env
         self.camera_id = camera_id
         self.viewer = None
         self.camera_config = cam_config
 
+        self.width = 1280
+        self.height = 800
+        self.sim = sim
+        self.camera_name = self.sim.model.camera_id2name(0)
+
     def render(self):
-        pass
+        # get frame with offscreen renderer (assumes that the renderer already exists)
```

Comment:
this render function is only supposed to be used for onscreen rendering. it's not supposed to return any values.